DAS/das/tests/mqtt_test.py

```diff
@@ -2,6 +2,8 @@
 import argparse
 import paho.mqtt.client as mqtt
 
+from mhp.topics import DAS, BOOST
```

Comment:
Is It possible to import these and rename as `DASTopics` and `BOOSTTopics` or similar? It would make them more readable and avoid misunderstanding when reading lines such as `DAS.start` (vs `DASTopics.start`), in my humble opinion.

Reply:
You've a very good point - I've had a think about this, and I totally agree withe that premise but I think lengthening the names would be redundant as the fact that it is a topic is made clear from the namespace. If we wanted to make the meaning of these more clear, we could instead import `from mhp import topics` and then call `topics.DAS.start` which is just as descriptive.

The question then becomes whether we should *actually* import like this instead. I feel like many would be resistant to it just because of the verbosity. I do agree with the points you've made (especially with regard to the naming collision with `WirelessModule` so I'll at least change it over in this repo.

Reply:
Tbh I actually like the idea of using `topics.DAS.start`. I don't personally think it's too verbose, it actually helps make the code more readable, for me at least.